docs/DeveloperGuide.md

```diff
@@ -9,15 +9,15 @@ title: Developer Guide
 
 ## **Acknowledgements**
 
-* Some code adapted from http://code.makery.ch/library/javafx-8-tutorial/ by Marco Jakob
+* Some code adapted from [Marco Jakob's JavaFX Tutorial](http://code.makery.ch/library/javafx-8-tutorial/) 
 
-* Some code inspired by https://github.com/se-edu/addressbook-level4 by CS2103T Teaching Team
+* Some code inspired by [SE-EDU AddressBook Level 4](https://github.com/se-edu/addressbook-level4)
 
-* Copyright by Susumu Yoshida - http://www.mcdodesign.com/
-  * address_book_32.png
+* Copyright by Susumu Yoshida - [Mcdodesign](http://www.mcdodesign.com/)
+  * address_book_32
```

Comment:
Would it be better to keep .png? Feels a bit confusing to just put `address_book_32` and `calendar`

Reply:
okay changed!